class-lifterlms.php

```diff
@@ -95,6 +96,7 @@ private function __construct() {
 		add_action( 'init', array( $this, 'processors' ), 5 );
 		add_action( 'init', array( $this, 'events' ), 5 );
 		add_action( 'init', array( $this, 'init_session' ), 6 ); // After table installation which happens at init 5.
+		add_action( 'init', array( $this, 'payment_gateways' ) );
```

Comment:
Nitpicking here. Since the other action callbacks are registered with a specific priority, and sorted by that priority, it would be nice if the other two `init` callbacks were registered with an explicit priority of 10. Yes, we know the default is 10, but someone new to WordPress may not.

Reply:
I'm going to leave this because this has no bearing on the bugfix.

It's more of a question of coding standards. And the question, to me, doesn't have much to do with hook callback priority so much as the usage of default values.

If we assume that new developers aren't familiar with WP function defaults we should probably assume that they're not familiar with our internal APIs defaults and then... should we ever use default values or should we always be explicitly declaring the values regardless of the ability to use defaults. And then should we *never code functions with default values*? 

The implications of making this kind of change for this reason are far-reaching. And don't concern this PR.